### ISSUE TITLE
.gitlab-cy.yml missing in pdk-templates 3.0? 

### ISSUE THREAD

**Issue by drakpzone** [puppetlabs/pdk-templates#524]
## Use Case
previous versions of PDK and pdk-templates used to provide a minimal (albeit functional) .gitlab-ci.yml, but apparently that's not the case with pdk templates v3.0. I was also hoping that a pdk upgrade would upgrade the existing .gitlab-ci.yml and drop the usage of puppet 6 for CI.

## Describe the Solution You Would Like
Get back previous .gitlab-ci.yml with puppet 7 and 8 as default

**Comment by cruelsmith** [comment]
Mhm, it has been removed with #510 since puppetlabs do not use it. But i would had been expected that this would have been at least mentioned in the release notes for pdk.

Im also now need to manually maintain the `.gitlab-ci.yml` for my gitlab projects :cry:

**Comment by silug** [comment]
> Mhm, it has been removed with #510 since puppetlabs do not use it. But i would had been expected that this would have been at least mentioned in the release notes for pdk.

I agree.  It's really odd that a large change like that was made with no mention in the release notes.

**Comment by Tamerz** [comment]
I would also like this back if possible.

**Comment by cruelsmith** [comment]
JFYI: Here is my updated version of `.gitlab-ci.yaml` for PDK 3 feel free to use and/or to readd it here again. https://gist.github.com/cruelsmith/3a5214730d1abb1ec4ebd3d01215c89f

**Comment by mattatnersc** [comment]
I understand why they would remove it (it is hard to maintain something you don't use), but our org uses gitlab, so I'd like it back.

**Comment by pmcmaw** [comment]
Hey everyone we are listening to you all and your input on the following discussion would be invaluable! https://github.com/puppetlabs/pdk-templates/discussions/562

**Comment by cdenneen** [comment]
@pmcmaw @david22swan discussion was forked and never had team provide interaction with... I just updated the thread:

> @pmcmaw seems like this discussion you requested died... it would be nice if team won't be adding back into moduleroot to allow some sort of key in .sync.yml to allow for custom file templates to be added/included... so people could continue to update the template-ref but keep a custom erb for their own files that might be necessary... it would help in custom scenarios where people are asking for things like gitlab-ci, CLAUDE, AGENTS, etc but allow them to do so in a custom way without having to keep forking pdk-templates and trying to keep it updated between custom and upstream changes... like a custom erb list or something or allow for having 2 pdk sync sources... so pdk-templates for the majority and custom-template for additional erb addons?

consider the idea of either custom erb addons or additional template-url ability? So we could sync pdk-templates and then custom ones as well? would `pdk sync` work if done twice with 2 different template-url and template-refs? So they don't step on each other?
